src/browser/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
```

Comment:
Check if svgs as component dependencies have @typed that we could use

Reply:
Looking into types for react-svg-inline, didn't seem to solve the problem. I think this way works well enough. React-svg-inline typed it's icon-prop to be a string.